client/src/modules/user/UserProfile/DeleteAccount.jsx

```diff
@@ -130,12 +162,22 @@ class DeleteAccount extends Component {
           onCancel={this.hideDeleteDialog}
           onConfirm={this.handleDeleteAccount}
           onEmailChange={this.handleEmailChange}
-          onlyOwnerResources={onlyOwnerResources}
           onPasswordChange={this.handlePasswordChange}
           onSubmit={this.handleDeleteAccount}
           onVerificationTextChange={this.handleVerificationText}
           pending={pending}
         />
+        <Dialog
+          cancelLabel={formatMessage({ id: 'common.button.cancel' })}
+          isVisible={isSelectionDialogVisible}
```

Comment:
change naming for this dialog - it is about ownership transfer so maybe OwnershipTransferInfoDialog or sth similar :) - it has nothing to do with selection.